users.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
Is this json tested? I doubt ...

Reply:
I did try editing the user details and saving the same, I was able to see it working, but I haven't  done a lot of tests

Reply:
Save will happen ...
Try to open the dashboard...
Check the flow you'll figure out it will break